packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/drp/configuration/common/mainPcc/translations/Messages_fr_FR.json

```diff
@@ -1,6 +1,8 @@
 {
-  "dedicatedCloud_datacenter_pcc_name": "Nom du Private Cloud : {{pcc}}",
-  "dedicatedCloud_datacenter_main_datacenter_name": "Identifiant du datacenter Private Cloud correspondant",
+  "dedicatedCloud_datacenter_pcc_name_EPCC": "Nom du Private Cloud : {{pcc}}",
+  "dedicatedCloud_datacenter_pcc_name_MBM": "Nom du Essentials : {{pcc}}",
```

Comment:
```suggestion
  "dedicatedCloud_datacenter_pcc_name_MBM": "Nom du Managed Bare Metal : {{pcc}}",
```
as it's the way it's called elsewhere

Reply:
According to specs, `Private Cloud` should be replaced by `Essentials` for a Managed Bare Metal PCC

Reply:
Maybe we can uniformize the naming because in some sections it is called "Managed Bare Metal", for example here : https://github.com/ovh/manager/pull/3915/files#diff-2cad5cf9c5b8653390ec5d4d1c18cd8b148a0231e9cb56c07ee4e627039016d1R13

Reply:
Yeah... In some places, we need the name to be shown as 'Essentials'. The specs does not promote uniform naming. But in general, while replacing the old names, Private Cloud must be changed to Essentials, according to specs and OVH Private Cloud by OVH Managed Bare Metal. It is a case by case naming and cannot be uniformized, if we go by the specs